atom/common/api/electron_bindings.cc

```diff
@@ -29,6 +29,7 @@
 #include "native_mate/dictionary.h"
 #include "services/resource_coordinator/public/cpp/memory_instrumentation/global_memory_dump.h"
 #include "services/resource_coordinator/public/cpp/memory_instrumentation/memory_instrumentation.h"
+#include "third_party/blink/renderer/platform/heap/process_heap.h"  // nogncheck
```

Comment:
@nornagon 
Is it ok to use `nogncheck` to suppress gn error `Can't include this header from here.`?

Reply:
we do it here as well https://github.com/electron/electron/blob/master/atom/renderer/renderer_client_base.cc#L38